### PR TITLE
Make done ratio disappear

### PR DESCRIPTION
**Pull request by Azure7111**
This is a fix for a regression found when testing https://community.openproject.org/work_packages/18712

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://azure7111-fix-make-done-ratio-disappear.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://azure7111-fix-make-done-ratio-disappear.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by myabc** [comment]
@Azure7111 should we not have a test for this?

**Comment by Azure7111** [comment]
@myabc Of course! I shall add one.

File: lib/api/v3/work_packages/form/work_package_payload_representer.rb [hunk 1]
```diff
@@ -77,7 +77,8 @@ def initialize(represented, options = {})
           property :done_ratio,
                    as: :percentageDone,
                    getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
```
Comment (myabc) [inline lib/api/v3/work_packages/form/work_package_payload_representer.rb:79]: @Azure7111 is the `if` statement here not redundant?

Comment (Azure7111) [inline lib/api/v3/work_packages/form/work_package_payload_representer.rb:79]: @myabc You are right, now that it doesn't show anymore when disabled I can probably remove the whole getter part.

**Pull request merged**